display-brightness-ddcutil@themightydeity.github.com/indicator.js

```diff
@@ -269,7 +268,7 @@ export const SingleMonitorSliderAndValueForStatusAreaMenu = class SingleMonitorS
             clearTimeout(this._timer);
     }
 
-    destroy() {
+    destory() {
```

Comment:
No sure why changing this to de-story, fixes Quicksetting slider not destorying when extension is disabled on when switch to systemtray panel mode.

Reply:
My guess here is that it may be a reserved word or something similar, but, honestly, I don't know much about GJS.